solc/CommandLineInterface.cpp

```diff
@@ -790,7 +799,7 @@ void CommandLineInterface::handleCombinedJSON()
 	{
 		Json::Value contractData(Json::objectValue);
 		if (requests.count(g_strAbi))
-			contractData[g_strAbi] = dev::jsonCompactPrint(m_compiler->contractABI(contractName));
```

Comment:
Sounds good. I was gonna make another change. The output of `solc --abi` is a file called `contractName.abi`

What do you think about changing that to `contractName.abi.json` and give the option to `--pretty-json` print it or compact?